modules/ducktests/tests/ignitetest/tests/pme_free_switch_test.py

```diff
@@ -81,7 +81,8 @@ def test(self, ignite_version):
 
         ignites.stop_node(ignites.nodes[self.NUM_NODES - 1])
 
-        long_tx_streamer.await_event("Node left topology", 60, from_the_beginning=True)
+        long_tx_streamer.await_event("Node left topology", 120 if ignite_version < V_2_8_0 else 60,
```

Comment:
This will not work, should be `IgniteVersion(ignite_version) < V_2_8_0`. And I suppose, that this should be 
 `IgniteVersion(ignite_version) > V_2_8_1`, isnt' it?

Reply:
Yep. Missed this in hurry. Fixed.